### PR TITLE
feat: added attendee access typing and session mapping

### PR DESCRIPTION
**Pull request by fletchthefletch**

### CODE REVIEW

File: resource/research2022-test/session_map.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "SessionSetMap": [
```
Comment (fletchthefletch) [inline resource/research2022-test/session_map.json:2]: @nelson-w I have made the assumption that test files should have the same data as prod files... is this assumption incorrect?

File: resource/research2022-prod/session_map.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "SessionSetMap": [
+    {
+      "HubbId": 1001,
+      "SessionTypeName": "Breakout",
+      "SessionSets": ["Session Catalog", "Session Scheduler", "My Schedule"]
+    },
+    {
+      "HubbId": 1002,
+      "SessionTypeName": "Plenary",
```
Comment (fletchthefletch) [inline resource/research2022-prod/session_map.json:10]: @nelson-w I haven't seen this session type elsewhere in our solution, but it was this session type name given in the access grid - am I correct in thinking this should be a new session type?

File: resource/research2022-prod/rules.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        "MyIgnite": true,
+        "ScheduleBuilder": true,
+        "MobileApp": true,
+        "MeetingScheduler": true,
+        "MobileAppMessaging": true,
+        "LocationBasedMessaging": true,
+        "InPersonExperience": true,
+        "PlatinumClub": false
+      },
+      "VisibleAttendeeTypes": ["Attendee"],
```
Comment (fletchthefletch) [inline resource/research2022-prod/rules.json:42]: @nelson-w curious about what this flag is for - does it just specify whether other attendees (with this type) can be searched for by other attendees (of this type)?

File: resource/research2022-prod/session_map.json (hunk 2 above)
Comment (nelson-w) [reply resource/research2022-prod/session_map.json:10]: Yes that's correct.

File: resource/research2022-test/session_map.json (hunk 1 above)
Comment (nelson-w) [reply resource/research2022-test/session_map.json:2]: Yes

File: resource/research2022-prod/rules.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      "SessionSetAccess": [
+        {
+          "ApplicableFrom": null,
+          "ApplicableTo": null,
+          "Identifier": "Session Catalog"
+        }
+      ]
+    },
+    {
+      "Identifiers": ["Attendee"],
```
Comment (nelson-w) [inline resource/research2022-prod/rules.json:30]: Update "attendee" to "General Attendee"

File: resource/research2022-prod/rules.json (hunk 3 above)
Comment (nelson-w) [reply resource/research2022-prod/rules.json:42]: Yes that's right.

File: resource/research2022-test/rules.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      "SessionSetAccess": [
+        {
+          "ApplicableFrom": null,
+          "ApplicableTo": null,
+          "Identifier": "Session Catalog"
+        }
+      ]
+    },
+    {
+      "Identifiers": ["Attendee"],
```
Comment (nelson-w) [inline resource/research2022-test/rules.json:30]: This also to "General Attendee"

Comment (fletchthefletch) [reply resource/research2022-test/rules.json]: Oops! Good catch

**Pull request merged**